src/Homepage.js

```diff
@@ -1,5 +1,9 @@
 import React, {Component} from 'react';
 
+const SwipePage = () => {
+    //Swipe
+}
+
 class Homepage extends Component { 
 
     constructor(props) 
```

Comment:
Is there a reason why you're using classes? If you saw the lecture you know I don't really like them in new code ;)